app.rb

```diff
@@ -0,0 +1,81 @@
+require_relative 'book'
+require_relative 'classroom'
+require_relative 'person'
+require_relative 'rental'
+require_relative 'student'
+require_relative 'teacher'
+
+class App
+  def initialize
+    @books = []
+    @people = []
+    @rentals = []
+    @classrooms = []
+  end
+
+  def list_all_books
+    @books.each do |book|
+      puts "Title: #{book.title}, Author: #{book.author}"
+    end
+  end
+
+  def list_all_people
+    @people.each do |person|
+      puts "ID: #{person.id}, Name: #{person.name}, Age: #{person.age}"
+    end
+  end
```

Comment:
- [ ] Good job so far but when listing people should display differently, it should be precise if the person is a student or a teacher, kindly fix that 🙏🏾 

Yours

<img width="696" alt="Screenshot 2023-07-27 at 21 41 54" src="https://github.com/Jrmcuervo/OPP-school-library/assets/74501523/9b166a1d-90ae-4322-8fd3-1c583ed795f6">

In the original video

<img width="696" alt="Screenshot 2023-07-27 at 21 42 10" src="https://github.com/Jrmcuervo/OPP-school-library/assets/74501523/b02a02e6-c32d-422b-9492-205a55cf3baa">